specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-03-01-preview/subscriptions.json

```diff
@@ -0,0 +1,169 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-03-01-preview",
+    "title": "SubscriptionClient",
+    "description": "Subscription client provides an interface to create and manage Azure subscriptions programmatically."
+  },
+  "schemes": [
+    "https"
+  ],
+  "host": "management.azure.com",
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Subscription/subscriptions/{subscriptionId}/cancel": {
+      "post": {
+        "operationId": "CancelSubscription",
```

Comment:
Please provide x-ms-examples describing minimum/maximum property set for response/request payloads for operations.

Same for all the operations.

Reply:
See https://github.com/Azure/azure-rest-api-specs/tree/16967289841f4f09aff7101448d8d45ca7f31681/specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/examples

Reply:
Added the x-ms-examples